src/components/Profile/My posts/MyPosts.jsx

```diff
@@ -5,7 +5,7 @@ import { textColorBlue } from "../../../assets/colors/colors";
 import { text_22, heading_30 } from "../../../assets/fonts/fonts";
 import MyPostForm from "./MyPostForm";
 
-const MyPosts = ({ posts, addPost }) => {
+const MyPosts = React.memo(({ posts, addPost }) => {
   const getPostElements = () => {
```

Comment:
I think that you can use just an assignment like this:
`const postElements = posts.map((post) =>...`
And in the return:
`<StyledText>{postElements}</StyledText>`